city_scrapers/spiders/bisnd_bcpc.py

```diff
@@ -0,0 +1,106 @@
+from urllib.parse import urljoin
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class BisndBcpcSpider(CityScrapersSpider):
+    name = "bisnd_bcpc"
+    agency = "Burleigh County Planning Commission"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.burleigh.gov/government/boardscommittees/planning-zoning-commission/"  # noqa
+    ]
+    location = {
+        "name": "Tom Baker Room",
+        "address": "City/County Building, 221 N 5th St, Bismarck",
+    }
+
+    def parse(self, response):
+        """
+        Parse meeting items from agency website.
+        """
+        time = response.css(".tbltitle p::text").get()
+        link1 = response.css(".info p")[6].css("a::attr(href)").get()
+        link2 = response.css(".info p")[7].css("a::attr(href)").get()
+        title1 = response.css(".info p::text")[7].get()
+        title2 = response.css(".info p::text")[8].get()
+        links = [
+            {"title": title1.replace("\xa0", " "), "href": link1},
+            {"title": title2, "href": link2},
+        ]
```

Comment:
**praise:** It's always great to include agendas and minutes in our returned data when they're available and easy enough to parse. Thanks for getting these.